### DISCUSSION TITLE
Establish a Mojo Improvement Proposal process

### DISCUSSION THREAD

**Discussion by crisadamo** [modular/modular#198] [Ideas]
Hi all,
Even though Mojo it's quite new and under heavily development and this will be important to have a sort of `Mojo Improvement Proposal` (MIP) Process, similar to whats happen on other languages. Such that, the community can provide the right set of details to discuss changes/features to potentially add to the languages as it evolves.

**Comment by TZDak** [comment]
I like it. I had suggested something similar, as part of a larger suggestion/discussion, but I can't find it anymore.

**Comment by goldiegadde** [comment]
Thanks for the suggestion @crisadamo and @TZDak , we will look into it and get back to you as soon as possible.
